src/redux/actions/pokemonCardActions.js

```diff
@@ -0,0 +1,67 @@
+import { fetchPokemon, fetchSpecies } from '../../utils';
+export const FETCH_CURRENT_POKEMON_REQUEST = 'FETCH_CURRENT_POKEMON_REQUEST';
+export const FETCH_CURRENT_POKEMON_SUCCESS = 'FETCH_CURRENT_POKEMON_SUCCESS';
+export const FETCH_CURRENT_POKEMON_ERROR = 'FETCH_CURRENT_POKEMON_ERROR';
+export const FETCH_CURRENT_SPECIES_REQUEST = 'FETCH_CURRENT_SPECIES_REQUEST';
+export const FETCH_CURRENT_SPECIES_SUCCESS = 'FETCH_CURRENT_SPECIES_SUCCESS';
+export const FETCH_CURRENT_SPECIES_ERROR = 'FETCH_CURRENT_SPECIES_ERROR';
+export const SHOW_CURRENT_POKEMON_MODAL = 'SHOW_CURRENT_POKEMON_MODAL';
+export const IS_COMPARING = 'IS_COMPARING';
+
+export const fetchCurrentPokemon = (url) => async (dispatch) => {
+  dispatch({
+    type: FETCH_CURRENT_POKEMON_REQUEST
+  })
+  const { payload } = await fetchPokemon(url);
```

Comment:
awaits can fail throwing an error when there are unexpected errors, also use the try catch to dispatch the error action or use .catch in the promise

Reply:
This comment applies for all other async await usages